### ISSUE TITLE
Review object and type documentation to ensure they correctly match the JSON schema

### ISSUE THREAD

**Issue by david-waltermire-nist** [usnistgov/vulntology#89]
# User Story:

As a Vulntology user, I need accurate documentation that reflects the model enforced by the JSON schema.

## Goals:

- [ ] Review all objects for consistency (i.e. correct relationships, properties, and cardinalities)
- [ ] Review all types for consistency, ensuring are valid values are reflected in both the schema and documentation

Non-goal: Don't check graphics. These will be updated later in a single pass once the model settles down.

Completing a successful review and addressing any identified problems will resolve #74.

## Dependencies:

Do this after #90 is closed.

## Acceptance Criteria

- [ ] All readme documentation affected by the changes in this issue have been updated.
- [ ] A Pull Request (PR) is submitted that addresses the goals of this User Story. This issue is referenced in the PR. If the PR only partially addresses a given User Story, the specific goals addressed are identified in the PR.

**Comment by Chris-Turner-NIST** [comment]
Note: Ensure MiTM is consistently referred to as Meddler-in-the-Middle across all documentation and visuals
